src/main/java/liquibase/ext/snowflake/database/SnowflakeDatabase.java

```diff
@@ -84,22 +84,22 @@ public String getDefaultDriver(String url) {
 
     @Override
     public String getDefaultCatalogName() {
-        return super.getDefaultCatalogName() == null ? null : super.getDefaultCatalogName().toUpperCase();
```

Comment:
I'm unsure on whether pulling the the toUpperCase() on the schema name stuff is going to cause problems or not. 

I'm going to look into that a bit

Reply:
It does not cause problems, but with the toUpperCase() removed, it's better to completely remove the following functions from this class (so it just calls the superclass version):

- getDefaultCatalogName
- getDefaultSchemaName
- getJdbcCatalogName
- getJdbcSchemaName
- isSystemObject (not because of this change, but because was already unneeded)